src/pages/Page404/Page404.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
The styling is not correct. On desktop it looks like this:
![Screenshot 2024-09-29 at 10 49 23](https://github.com/user-attachments/assets/58ec5eac-19b8-4d8f-a65f-f826a0af5c71)
On mobile it looks like this:
![Screenshot 2024-09-29 at 10 49 57](https://github.com/user-attachments/assets/f64d1eb5-4863-4f2c-90e0-de886f893602)

Also we do not want to show the header on the 404 screen